docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc

```diff
@@ -24,6 +25,15 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.15.4 relnotes
+
+[[release-notes-8.15.4]]
+== {fleet} and {agent} 8.15.4
+
+There are no bug fixes for {fleet}, {fleet-server}, or {agent} in this release.
```

Comment:
```suggestion
[discrete]
[[bug-fixes-8.15.3]]
=== Bug fixes

{agent}::
* Improve upgrade experience by assuming that {agent} is installed, allowing for the control path to be set correctly. {agent-pull}5955[#5955]
```

Reply:
@pierrehilbert, please let me know what you think.

Reply:
allowing for control path to be set correctly or maybe "to use proper control path to communicate with running agent" alternatively

Reply:
I like your suggestion @michalpristas thanks.